server/src/database/reactions.go

```diff
@@ -0,0 +1,88 @@
+package database
+
+import (
+	"context"
+	"github.com/google/uuid"
+	"github.com/uptrace/bun"
+	"scrumlr.io/server/common"
+)
+
+type Reaction struct {
+	bun.BaseModel `bun:"table:reactions"`
+	ID            uuid.UUID
+	Note          uuid.UUID
+	User          uuid.UUID
+	ReactionType  string
```

Comment:
Like already mentioned for the reactions DTO, this should be something like `types.ReactionType` instead of `string`